assets/css/omise-css.css

```diff
@@ -234,6 +234,14 @@ ul.omise-banks-list {
 	border: 4px solid #fff;
 }
 
+.omise-banks-list .mobile-banking-logo {
```

Comment:
on the screenshots you attached, the logo and label look slightly off from the checkbox. Is this something we can fix? Or maybe is this the same as installments too?